src/main/pages/che-7/hands-on-guidance/proc_configuring-a-che-workspace-using-a-devfile.adoc

```diff
@@ -1,27 +1,87 @@
 [id="configuring-a-che-workspace-using-a-devfile_{context}"]
 = Configuring a Che workspace using a Devfile
 
-This paragraph is the procedure module introduction: a short description of the procedure.
+This section will show how to select a workspace, loading in the required stack and a sample project, all in one step.
+
+Please refer to the link:using-developer-environments-workspaces.html#making-a-workspace-portable-using-a-devfile_using-developer-environments-workspaces[Devfile user guide] and link:https://redhat-developer.github.io/devfile/devfile[Devfile specifications] for detailed information how to add or remove components, or how to set memory and environment variables.
 
 [discrete]
 == Prerequisites
 
-* A bulleted list of conditions that must be satisfied before the user starts following this assembly.
-* You can also link to other modules or assemblies the user must follow before starting this assembly.
-* Delete the section title and bullets if the assembly has no prerequisites.
+* You need an OKD, OCP, or OSD cluster, if using Che on Openshift. See below for use of link:https://www.okd.io/minishift/[Minishift].
+* Or, you need a Kubernetes cluster such as link:https://github.com/kubernetes/minikube#installation[Minikube] to use Che on Kubernetes.
 
 [discrete]
 == Procedure
 
-. Start each step with an active verb.
+=== Importing a project and workspace in one step using a devfile
+
+In Che 7, stacks are defined as link:using-developer-environments-workspaces.html#making-a-workspace-portable-using-a-devfile_using-developer-environments-workspaces[devfiles], instead of container images. This affords more flexibility when defining the project(s), IDE, commands, tools, and application runtimes needed in your workspace.
+
```

Comment:
you have "the user" and "your". Pick just one person - 3rd or 2nd, but not both. :)

Reply:
+1

(Pick the 'user'. Docs should never address the reader directly. Not only does it sound bad, but it's also incorrect because the reader may not be the person who's actually doing the stuff.)

Reply:
Sure, @rkratky .